src/framework/mlt_playlist.c

```diff
@@ -1819,13 +1819,17 @@ void mlt_playlist_pad_blanks( mlt_playlist self, mlt_position position, int leng
 }
 
 /** Insert a clip at a specific time.
+ * If the insertion is done in overwrite mode (mode=true):
+ *      - if the position is a blank then it is replaced by the clip. If the blank was shorter than the clip, then the following clip is pushed to make room for the new clip.
+        - if the position is inside a clip, then the new clip is inserted before or after the old one, depending on whether the position is past the middle of the old clip or not. In case it is inserted after, if there is some blank after, it is replaced by the clip similarly as before
+ * If the insertion is not done in overwrite mode, then the behaviour is the same except that blanks are considered as normal clips (and thus never deleted).
```

Comment:
There should be a blank comment line between the doxygen brief doc and the extended description. The doxygen description should be hard wrapped at 80 columns.